### DISCUSSION TITLE
How does Hubs Cloud connect with Mozilla?

### DISCUSSION THREAD

**Discussion by PaintedShepherd** [mozilla/hubs#3807] [Hubs Cloud Questions] [answered]
I want to use the hubs cloud for some professional projects, but i'm not sure, what happens, when mozilla shuts down the hubs cloud servers. Am i safe, when i host my own hubs cloud?
Or is there still a connection to mozillas own servers, so my project will be no longer available?

**Comment by robinkwilson** [comment, marked as answer]
Hubs Cloud was specifically designed so if you deploy a Hubs Cloud instance, you are using your own servers and run the software independently regardless of Mozilla's status. The only connection your Hubs Cloud instance has to Mozilla is the automatic update cycle we do every month or two that pulls in the latest changes and updates to Hubs. If that automatic update cycle were to stop, you'd still be able to use your Hubs Cloud instance normally.

In other words, yes you are safe to host your own hubs cloud :)

The service that **is** hosted by Mozilla is hubs.mozilla.com so that one is controlled by Mozilla. Hubs Cloud is not.
